packages/components/package.json

```diff
@@ -40,8 +40,7 @@
     "build:global-styles": "postcss ./styles/global.css --output dist/global.css",
     "build:combine-styles": "concat-cli -f ./dist/global.css ./dist/styles.css -o ./dist/styles.css && rm ./dist/global.css",
     "build:styles": "pnpm build:global-styles && pnpm build:combine-styles",
-    "test": "FORCE_COLOR=1 jest",
```

Comment:
FYI this `FORCE_COLOR=1` is for us to see colour in the terminal when we use turbo to run these commands (colour gets stripped otherwise). Maybe just check if it's still relevant or not :)

Reply:
Doesn't make a difference any more :'D'